doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+* is **Object oriented**. Typical user-exposed functions don't operate on raw data matrices, but on `Raw`, `Epochs`, `Evoked`, ... objects. Many object methods operate in-place. For example, when in MNE, you do EEG = pop_eegfiltnew(EEG, ...), in MNE, you do raw.filter(...). In EEGLAB, you do pop_plot_topo(EEG, ...). In MNE, you do evoked.plot_topomap(...). In EEGLAB, you do pop_saveset(EEG, ...). In MNE, you do `epochs.save(...)`. `raw.plot()` is very different from `evoked.plot()`. In-place modification can usually be turned off with the `copy=True` setting.
+* does **not feature a GUI**. MNE always requires explicit code. This can be frustrating for beginners, but in the long run, it supports good scientific practices - better reproducible, more scalable analyses. Generally speaking, there is some trade-off between the lack of GUI in MNE and the Python base; while GUIs are more accessible to newcomers, Python more easily allows experienced users to create custom and advanced analysis pipelines and techniques.
+* has a different *status and style* of the project. EEGLAB is the biggest EEG analysis package. MNE is a comparatively young project. It follows modern coding standards (including code review and unit tests). Its development is less monolithic, more distributed and democratic than that of EEGLAB. There are many published papers using EEGLAB, and very few with MNE-Python, and many users of EEGLAB and few of MNE.
+
+Minor differences: MNE-Python ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* is much more developed with regards to **MVPA/decoding analyses**. It is also developed with a close eye on the leading Machine Learning toolbox *scikit-learn*.
+* 's *ICA is not as deeply integrated* into the system. Support for clustering ICs is very rudimentary. However, MNE's ICA implementations are fully featured, and in some ways superior to EEGLAB. For example, MNE's infomax implementation can be much faster than EEGLAB's, in large part due to multicore usage. Also, MNE ICA objects allow easily applying the same ICA fit to multiple data sets (e.g., it is easy to fit ICA on high pass filtered data and apply it to unfiltered data).
+* does not feature *plugins* as strongly as EEGLAB does. No true equivalents exist for high quality plugins such as LIMO or ERPLAB. Some features which are plugins in EEGLAB are parts of base MNE, such as CORRMAP, and many plugins are at least partially mirrored by MNE core functionality (although ironically, not Alex Gramfort's graph cut plugin for single trial analysis!!!).
+* does not have as broad *data system format support* for EEG data formats as EEGLAB does with `fileio`. However, MNE can read many common file formats, including the EEGLAB format, and of course many MEG formats.
```

Comment:
ok got it but I would rephrase.

does not have an extensive support of the various EEG *data formats* as EEGLAB has with `fileio`. However, MNE can read many common file formats, including the EEGLAB format, and of course many MEG formats.